packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -180,4 +180,92 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
       type: retrieveType,
     );
   }
+
+  @override
+  Future<XFile?> getImage({
+    required ImageSource source,
+    double? maxWidth,
+    double? maxHeight,
+    int? imageQuality,
+    CameraDevice preferredCameraDevice = CameraDevice.rear,
+  }) async {
+    String? path = await _getImagePath(
+      source: source,
+      maxWidth: maxWidth,
+      maxHeight: maxHeight,
+      imageQuality: imageQuality,
+      preferredCameraDevice: preferredCameraDevice,
+    );
+    return path != null ? XFile(path) : null;
+  }
+
+  @override
+  Future<List<XFile>?> getMultiImage({
+    double? maxWidth,
+    double? maxHeight,
+    int? imageQuality,
+  }) async {
+    final List<dynamic>? paths = await _getMultiImagePath(
+      maxWidth: maxWidth,
+      maxHeight: maxHeight,
+      imageQuality: imageQuality,
+    );
+    if (paths == null) return null;
+
+    final List<XFile> files = [];
+    for (final path in paths) {
+      files.add(XFile(path));
+    }
+    return files;
```

Comment:
This can be rewritten as:

```dart
return paths.map((String path) => XFile(path)).toList();
```

(especially since we're not doing any validation of `path` within the `for` loop?)

Reply:
Yep, agreed. Came from the existing function as well, fixed it in both instances.